include/sound/sof/channel_map.h

```diff
@@ -0,0 +1,65 @@
+/* SPDX-License-Identifier: BSD-3-Clause
```

Comment:
for kernel stuff, the license is GPLv2 or BSD-3-clause, see other files in include/sound/sof

Reply:
**please fix the license!!!!!**